_api-files/embedding-api/documentation/quick-start.md

```diff
@@ -0,0 +1,282 @@
+
+      When [creating a source](#create-a-source), you can choose to include the [source's form properties](##source-form-properties) in the `properties` argument. This information can be retrieved using the Source Type endpoint.
+
+      After a source's form is created, the `report_card` object within the source should be used to complete its configuration. The [Report Card](#report-card-object) object provides information about the steps required to configure the connection, their sequence, and the progress towards completing the steps.
+
+      In the sections below, we'll demonstrate two methods for creating a HubSpot source.
+
+    subsections:
+      - title: "Method 1: Use the Source Types Endpoint to Provide Properties "
+        anchor: "quick-start--source-creation-method-1"
+        content: "In this example, we'll use the Source Types endpoint to retrieve the source form properties for HubSpot, which has a `type` of `platform.hubspot`."
+
+        steps:
+          - title: "Retrieve HubSpot's Report Card"
+            anchor: "quick-start--source-creation-method-1--step-1"
+            content: |
+              Using the Source Types endpoint, retrieve the report card for `platform.hubspot`:
+
+              ```curl
+              curl -X GET {{ page.api-base-url}}/v4/source-types/platform.hubspot
+                   -H 'Authorization: Bearer <ACCESS_TOKEN>'
+              ```
+
+          - title: "Locate Required Properties for the HubSpot Form"
+            anchor: "quick-start--source-creation-method-1--step-2"
+            content: |
+              Use the response from Step 1 to locate the required properties for the `form` step. **Note**: You do not have to provide system-provided properties to create a source.
+
+              ```json
+              {  
+                 "type":"platform.hubspot",
+                 "current_step":1,
+                 "current_step_hints":{  
+                    "api":{  
+                       "method":"POST",
+                       "url":"/v4/sources"
+                    },
+                    "js":{  
+                       "function":"addSource",
+                       "options":{  
+                          "type":"platform.hubspot"
+                       }
+                    }
+                 },
+                 "steps":[  
+                    {  
+                       "type":"form",                                 // form step
+                       "properties":[
+                          {  
+                             "name":"image_version",                  // system-provided property
+                             "required_to_be_fully_configured":true,
```

Comment:
This property should be `is_required` everywhere.

Reply:
I'm not quite sure what you mean - can you clarify?  In examples where this property is listed, `required_to_be_fully_configured` is marked as `true`. Are you referring to something else?